core/src/main/kotlin/notification/Manager.kt

```diff
@@ -98,6 +98,8 @@ class Notifications(
 
     val valuesDesc get() = desc.values
 
+    val valuesAsc get() = asc.values
```

Comment:
remove this
methods should not be added just for unit tests
asc/desc can be tested via get() and contains()